README.md

```diff
@@ -3,3 +3,29 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/b13777f5d669412cb02635a66b5de4a8)](https://app.codacy.com/gh/BuildForSDG/farmrail?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDG/farmrail&utm_campaign=Badge_Grade_Dashboard)
 
 Build for SDG project repo
+## Project Name: Farm Rail
+
+PRODUCT DESCRIPTION
+
+Farmrail is a web-application that connects farmers to produce buyers/offtakers,
+Sellers of farm inputs like drugs, insecticides and other essential inputs,provides farmers with
+access to right information in regards to market variations and best farming practices.
+
+## Technology stacks used
+
+front-end
+ -React
+Backend 
+ -Python Django
+ 
+ ## file structure
+ 
+ frontend-files
+  src/Web_ui
+  
+ Backend-files
+ src/Services
+ 
+ ## installation
```

Comment:
Please reference to https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3 for a better Read Me